docs/resources/monitor.md

```diff
@@ -0,0 +1,192 @@
+---
+page_title: "Mackerel: mackerel_monitor"
+subcategory: "Monitors"
+description: |-
+
+---
+
+# Resource: mackerel_monitor
+
+This resource allows creating and management of monitor.
+
+## Example Usage
+
+### host_metric
+
+```terraform
+resource "mackerel_monitor" "cpu_high" {
+  name                  = "cpu % is high"
+  is_mute               = false
+  notification_interval = 10
+
+  host_metric {
+    metric   = "cpu%"
+    operator = ">"
+    warning  = 80
+    critical = 90
+    duration = 3
+
+    scopes = ["app", "proxy:ec2"]
+  }
+}
+```
+
+### connectivity
+
+```terraform
+resource "mackerel_monitor" "connectivity" {
+  name = "connectivity"
+
+  connectivity {
+    scopes = ["app", "proxy:ec2"]
+  }
+}
+```
+
+### service_metric
+
+```terraform
+resource "mackerel_monitor" "service_metric" {
+  name = "error rate is high"
+
+  service_metric {
+    service  = "app"
+    metric   = "execution.error.rate"
+    operator = ">"
+    duration = 3
+    warning  = 2
+  }
+}
+```
+
+### external
+
+```terraform
+resource "mackerel_monitor" "external" {
+  name                  = "Example Domain"
+  notification_interval = 10
+
+  external {
+    method                 = "GET"
+    url                    = "https://example.com"
+    service                = "app"
+    response_time_critical = 10000
+    response_time_warning  = 5000
+    response_time_duration = 3
+    headers                = { "name" : "Cache-Control", "value" : "no-cache" }
+  }
+}
+```
+
+### expression
+
+```terraform
+resource "mackerel_monitor" "role_avg" {
+  name = "role average"
+
+  expression {
+    expression = "avg(roleSlots(server:role,loadavg5)))"
```

Comment:
`service:role` ?

Reply:
I fix it https://github.com/mackerelio-labs/terraform-provider-mackerel/pull/51/commits/e0e3df33739decb81fef7f40543f6878d8073d0f . Thank you for your comment !